src/component/axis/AxisBuilder.ts

```diff
@@ -810,6 +810,15 @@ function buildAxisLabel(
         });
         textEl.anid = 'label_' + tickValue;
 
+        graphic.setTooltipConfig({
+            el: textEl,
+            componentModel: axisModel,
+            itemName: formattedLabel,
+            formatterParamsExtra: {
```

Comment:
I'm thinking if there is some way to allow the developer to detect if the label is overflowed and ellipsised to decide whether to show the tooltip and the content tp show.